tests/services/test_tasks_service.py

```diff
@@ -0,0 +1,115 @@
+from collections.abc import Sequence
+from unittest.mock import patch
+
+import pytest
+from tad.models import Status, Task, User
+from tad.repositories.statuses import StatusesRepository
+from tad.repositories.tasks import TasksRepository
+from tad.services.statuses import StatusesService
+from tad.services.tasks import TasksService
+
+
+class MockStatusesRepository:
+    def __init__(self):
+        self._statuses: list[Status] = []
+        self.reset()
+
+    def reset(self):
```

Comment:
this function does not exist in the StatusesRepository.

Reply:
Valid point. However, we are testing the service and not the repository and the reset is called by the test functions. This could be solved by testing everything in its own function, as the mockrepository is recreated each time then.